pro/src/screens/OfferEducational/__specs__/OfferEducationalEventAddressStep.spec.tsx

```diff
@@ -40,72 +28,60 @@ describe('screens | OfferEducational : event address step', () => {
+      ).not.toBeChecked()
+      expect(screen.getByLabelText('Autre')).not.toBeChecked()
+
+      expect(screen.getByLabelText('Sélectionner le lieu')).toHaveValue(
+        'VENUE_ID'
+      )
+      expect(screen.getByLabelText('Sélectionner le lieu')).toBeDisabled()
+
+      expect(
+        screen.getByText('Venue name', { exact: false, selector: 'p' })
+      ).toBeInTheDocument()
     })
 
     it('should display text area when user selects "other"', async () => {
       renderEACOfferForm(props)
 
-      // wait for page to be rendered
-      const offererSelect = queryOffererSelect()
-      await waitFor(() => expect(offererSelect.input).toBeInTheDocument())
-
-      const { offererVenueRadio, schoolRadio, otherRadio } =
-        queryOfferVenueRadioButtons()
+      await userEvent.click(await screen.findByLabelText('Autre'))
+      expect(screen.getByLabelText('Autre')).toBeChecked()
 
-      userEvent.click(otherRadio.input as HTMLInputElement)
+      expect(
+        screen.queryByLabelText('Sélectionner le lieu')
+      ).not.toBeInTheDocument()
 
-      const offerVenueSelect = queryOfferVenueSelect()
-      const offerVenueTextArea = queryOfferVenueTextArea()
-
-      expect(offererVenueRadio.input?.checked).toBe(false)
-      expect(schoolRadio.input?.checked).toBe(false)
-      expect(otherRadio.input?.checked).toBe(true)
-      expect(offerVenueSelect.input).not.toBeInTheDocument()
-      expect(offerVenueTextArea.input).toBeInTheDocument()
+      expect(
+        screen.getByLabelText('Adresse de l’événement')
+      ).toBeInTheDocument()
     })
 
     it('should not display neither event venue address nor text area if user selects "school"', async () => {
       renderEACOfferForm(props)
 
-      // wait for page to be rendered
-      const offererSelect = queryOffererSelect()
-      await waitFor(() => expect(offererSelect.input).toBeInTheDocument())
-
-      const { offererVenueRadio, schoolRadio, otherRadio } =
-        queryOfferVenueRadioButtons()
-
-      userEvent.click(schoolRadio.input as HTMLInputElement)
+      await userEvent.click(
+        await screen.findByLabelText('Dans l’établissement scolaire')
+      )
+      expect(
+        screen.getByLabelText('Dans l’établissement scolaire')
+      ).toBeChecked()
 
-      const offerVenueSelect = queryOfferVenueSelect()
-      const offerVenueTextArea = queryOfferVenueTextArea()
+      expect(
+        screen.queryByLabelText('Sélectionner le lieu')
+      ).not.toBeInTheDocument()
 
-      expect(offererVenueRadio.input?.checked).toBe(false)
-      expect(schoolRadio.input?.checked).toBe(true)
-      expect(otherRadio.input?.checked).toBe(false)
-      expect(offerVenueSelect.input).not.toBeInTheDocument()
-      expect(offerVenueTextArea.input).not.toBeInTheDocument()
+      expect(
+        screen.queryByLabelText('Adresse de l’événement')
+      ).not.toBeInTheDocument()
     })
   })
-
+  // TO DO: move this test, it does not belong to Address step
```

Comment:
```suggestion
  // TODO: move this test, it does not belong to Address step
```
C'est une note que tu pense fixer avant merge ou pour une prochaine étape ?